cardano-tx-submit/testing-tx-submit.md

```diff
@@ -0,0 +1,73 @@
+# Testing cardano-tx-submit-webapi
+
+Setting this up for testing and for actual use on a real network is significantly more difficult
+than it should be. This is definitely not ready for end users yet.
+
+The biggest thing missing at the moment is documentation of generation of transactions with the
+command line tools. That needs to be *thoroughly* documented and this document is not the place
+for that.
+
+The following instructions are for generating a testnet and testing tx-submission on that testnet.
+
+* This process generates a lot of state in the `cardano-node` checkout that can be cleaned up using:
+    ```
+    cardano-node $ rm -rf configuration/GenesisFiles/ db-* logs/ transaction.bin
+    ```
+
+* Generated the Genesis file for the testnet:
+    ```
+    cardano-node $ ./scripts/genesis.sh
+    ```
+  Part of the output of the above command is somethng like:
+    ```
+    genesis created with hash = 6f9371...939776
+      in directory .../cardano-node/configuration/GenesisFiles
+    ```
+
+* Run up a `tmux` session (the nodes of the testnet will end up in the tmux session):
+    ```
+    cardano-node $ tmux new-session -s Demo
+    ```
+
+* Run the testnet using the following. The `sleep` command is required to ensure sufficient time
+  has elapsed between the creationg of the genesis file and starting of the testnet.
+    ```
+    cardano-node $ sleep 100 ; ./scripts/shelley-testnet.sh
+    ```
+
+* Get a delegate key so that a transaction can be generated:
+    ```
+    cardano-node $ ./scripts/get-default-key-address.sh \
```

Comment:
This is also minor and hard to tell how many times it will change... but now the path looks different and it is `genesis` folder. At this moment I can't use this script I got error:  

> ./scripts/get-default-key-address.sh configuration/genesis/delegat-keys.001.key
./scripts/get-default-key-address.sh: 16: ./scripts/lib-node.sh: Syntax error: "(" unexpected

Reply:
Opened a ticket for that: https://github.com/input-output-hk/cardano-node/issues/411
If I will find time I will change the script to fix it.

Reply:
Yes, it will likely change again, but I have fixed it for now.